### PR TITLE
Completed issue-4-edit-review-Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
https://user-images.githubusercontent.com/97363006/182956028-2b166fbe-a85a-4efc-84d6-5b43248b0b3a.mp4

https://user-images.githubusercontent.com/97363006/182956033-b61f31fd-100c-4136-b6cb-20d520ed524e.mp4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -182,17 +182,40 @@ $form.addEventListener('submit', submitReview);
...
+    review.reviewId = data.editing.reviewId;
+
+    for (var i = 0; i < data.reviews.length; i++) {
+      if (data.reviews[i].reviewId === review.reviewId) {
+        data.reviews[i] = review;
+      }
+    }
+    var reviewItem = document.querySelectorAll('#review-item');
+    for (i = 0; i < reviewItem.length; i++) {
+      var reviewItemId = JSON.parse(reviewItem[i].getAttribute('data-review-id'));
```
Comment (codingcodymiller) [inline js/main.js]: You should use `parseInt`, not `JSON.parse`

File: js/main.js [hunk 2]
```diff
@@ -236,3 +270,29 @@ document.addEventListener('DOMContentLoaded', function (event) {
     $yourReviewsList.appendChild(reviewResult);
   }
 });
+
+$yourReviewsList.addEventListener('click', function (event) {
+  if (event.target && event.target.matches('BUTTON')) {
+    var $liClosest = event.target.closest('li');
+    var $reviewsId = $liClosest.getAttribute('data-review-id');
+
+    $reviewsId = JSON.parse($reviewsId);
```
Comment (codingcodymiller) [inline js/main.js]: You should use `parseInt`, not `JSON.parse`

**Review by codingcodymiller — APPROVED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Clicked approve by mistake, sorry!

File: js/main.js (hunk 1 above)
Comment (dulceherrera) [reply js/main.js]: Screen Shot 2022-08-04 at 2 59 25 PM

File: js/main.js (hunk 2 above)
Comment (dulceherrera) [reply js/main.js]: Screen Shot 2022-08-04 at 2 58 38 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**